wrappers/netcore/README.md

```diff
@@ -0,0 +1,19 @@
+# .NET Core wrapper example for 1DS C/C++ SDK
+
+Note that this wrapper is incomplete simple reference implementation that illustrates how to use 1DS C API from cross-platform .NET Core application.
```

Comment:
I'll change the wording of `incomplete` to `provides the same level of functionality as C API`, i.e. some features are accessible only from C++ layer. This still allows the native code initializer to use `SetContext(...)` efficiently, to populate common known fields, like `session id`, etc.. C# layer becomes `guest` to `host` initialized by native library.